RELEASE/scripts/autoscend.ash

```diff
@@ -2300,57 +2300,62 @@ boolean questOverride()
 
 boolean L13_towerNSNagamar()
 {
+	//status != 12 is an override in case auto_wandOfNagamar is incorrectly set to false
+	//status 11 means ready to fight sorceress, status 12 means you lost to her because you lacked a wand of nagamar
 	if (!get_property("auto_wandOfNagamar").to_boolean() && internalQuestStatus("questL13Final") != 12)
 	{
 		return false;
 	}
-
+	
 	if(item_amount($item[Wand of Nagamar]) > 0)
 	{
 		set_property("auto_wandOfNagamar", false);
 		return false;
 	}
-	else if (internalQuestStatus("questL13Final") == 12)
+	
+	if(creatable_amount($item[Wand Of Nagamar]) == 0 && (creatable_amount($item[WA]) > 0 || item_amount($item[WA]) > 0))
+	{	
+		pullXWhenHaveY($item[ND], 1, 0);
+	}
+	if(creatable_amount($item[Wand Of Nagamar]) > 0)
+	{
+		return create(1, $item[Wand Of Nagamar]);
+	}
+	
+	//hunt for bear verb orgy
+	if (item_amount($item[Wand of Nagamar]) == 0 && internalQuestStatus("questL13Final") == 12)
 	{
 		return autoAdv($location[The VERY Unquiet Garves]);
 	}
-	else if(pulls_remaining() >= 2)
+	
+	if(auto_my_path() == "G-Lover")
 	{
-		if((item_amount($item[ruby w]) > 0) && (item_amount($item[metallic A]) > 0))
-		{
-			cli_execute("make " + $item[WA]);
-		}
-		pullXWhenHaveY($item[WA], 1, 0);
-		pullXWhenHaveY($item[ND], 1, 0);
-		cli_execute("make " + $item[Wand Of Nagamar]);
-		return true;
+		pullXWhenHaveY($item[Ten-Leaf Clover], 1, 0);
 	}
 	else
 	{
-		if(auto_my_path() == "G-Lover")
+		pullXWhenHaveY($item[Disassembled Clover], 1, 0);
+	}
+	
+	if(cloversAvailable() > 0)
+	{
+		cloverUsageInit();
+		autoAdv($location[The Castle in the Clouds in the Sky (Basement)]);
+		cloverUsageFinish();
+		if(creatable_amount($item[Wand Of Nagamar]) > 0)
 		{
-			pullXWhenHaveY($item[Ten-Leaf Clover], 1, 0);
+			return create(1, $item[Wand Of Nagamar]);
 		}
 		else
 		{
-			pullXWhenHaveY($item[Disassembled Clover], 1, 0);
-		}
-		if(in_hardcore() && in_koe())
-		{
-			// TODO: Improve support
-			abort("In Kingdom of Exploathing: Please buy a Wand of Nagamar from the bazaar and re-run.");
-			return false;
+			auto_log_warning("Clovering [The Castle in the Clouds in the Sky (Basement)] for wand parts failed for some reason", "red");
```

Comment:
Is this abort-worthy?

Reply:
I agree.
But to clarify, I already removed the abort from koe (which was there because koe feature was unimplemented and required manual handling).

I have put in a cli log warning if it fails to clover, but it will not abort.